core/client/src/main/java/alluxio/client/file/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -32,8 +33,10 @@
  * to store the block, for the values mCapacityBytes minus mUsedBytes is not the available bytes.
  */
 @ThreadSafe
-public final class LocalFirstAvoidEvictionPolicy implements FileWriteLocationPolicy {
+public final class LocalFirstAvoidEvictionPolicy
```

Comment:
I would prefer separating the implementation of the two interfaces to two different classes so that all `FileWriteLocationPolicy` implementations are under `alluxio.client.file.policy` and all `BlockLocationPolicy` implementations are under under `alluxio.client.block.policy` even if it means some logic duplication ... I don't feel strongly about this though, so I leave it up to you